src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java

```diff
@@ -38,8 +38,9 @@ public static void setup() throws Exception {
     Provider.loadProviders(location, ProviderTest.providerRandom);
     module = new EncounterModule();
     // Ensure Person's Payer is not null.
-    Payer.loadNoInsurance();
-    person.coverage.setPayerAtTime(System.currentTimeMillis(), Payer.noInsurance);
+    PayerManager.loadPayers(new Location("Massachusetts", null));
```

Comment:
Ideally, you do not want to specify a location in the unit tests like "Massachusetts" because it will break synthea-international support. Instead, you want to do something like

```
String testStateDefault = Config.get("test_state.default", "Massachusetts");
PayerManager.loadPayers(new Location(testStateDefault, null));
```